### ISSUE TITLE
Admin UX tweak

### ISSUE THREAD

**Issue by btopro** [backdrop/backdrop-issues#970]
https://www.drupal.org/project/admin_menu_dropdown is worth a look as far as extending upon the current admin_menu pulled into backdrop core UX that's going on. It's a very simple project that gives the (configurable) option of hiding the admin menu when pressing `. This is huge when demoing to low-tech / excessively visual clients who get off-put by seeing what's under the hood.

**Comment by quicksketch** [comment]
This looks interesting. I think it may be useful to hide the admin menu in certain situations certainly (e.g. conflicting "sticky" navigation or when an anchor link scrolls to a part of the page covered by the admin menu). However at the same time I'm concerned about what would happen if a user accidentally hid the menu and didn't know how to get it back. Without the admin menu, many users may not know how to log out or get to administrative pages.

Overall, still interesting. I'd like to know what others think.

**Comment by btopro** [comment]
out of the box options are "hidden by default" and what key to press. So long as it's not hidden by default in the settings you have to actively press `. This isn't tracked page to page via cookie or anything though so it shows up next page. We've been using it on all builds now even just for screenshots of functionality.

**Comment by quicksketch** [comment]
Oh, well just an option that temporarily hides it doesn't seem like it would do much harm. If the worst-case scenario is a reload or visiting another page, that seems like it would mitigate pretty much any possible problem.

If you could indulge me, what happens when you type a backtick into a textarea? Does that hide the admin menu as well or does it have some kind of prevention when typing into an form input?

**Comment by btopro** [comment]
I don't recall but I believe it blocks it from typing if a form element is in scope, I rarely type those though so I might be wrong :)

**Comment by klonos** [comment]
> ...what happens when you type a backtick into a textarea?

![admin_menu_hide](https://cloud.githubusercontent.com/assets/2423362/8007849/d1a2ae0e-0be1-11e5-97af-43c6f789d46d.gif)

**Comment by klonos** [comment]
I don't really like the idea of such a thing in core, but that's only my personal opinion. If the reason is to get the menu out of the way for whatever purpose, then lets implement something that is visible/clickable. Like the count-down info bar over at simplytest.me for example:

![simplytest me_countdown_hide_show](https://cloud.githubusercontent.com/assets/2423362/8007967/806a4b54-0be2-11e5-863e-3ced9017b14a.gif)

**Comment by btopro** [comment]
@klonos agreed. that was more so the spirit of what I was going for using admin_menu_dropdown as a showcase of something to get feedback from; funny ur gif above w/ all the backticks didn't realize that. Going to reference this in the elmsln queue since we use that heavily.

**Comment by BWPanda** [comment]
I couldn't find another issue that dealt with the admin bar being toggle-able, so repurposing this issue to be a bit more generic. I also think maybe this should be a contrib module, then maybe in future we can merge it into core if there's enough demand.

**Comment by klonos** [comment]
> ...this should be a contrib module, then maybe in future we can merge it into core if there's enough demand.

👍 ...given that there were no further comments here since 2015, I think that leaving it to contrib.
